backend/views.py

```diff
@@ -1,8 +1,23 @@
-from canvas_oauth.oauth import get_oauth_token
+import logging
+
+from canvas_oauth.oauth import get_oauth_token, handle_missing_token
+from canvas_oauth.models import CanvasOAuth2Token
+from canvas_oauth.exceptions import InvalidOAuthReturnError
+
+from django.contrib.auth.models import User
 from django.contrib.auth.decorators import login_required
 from django.shortcuts import render
 
+
+logger = logging.getLogger(__name__)
+
 @login_required
 def get_home_template(request):
-    access_token = get_oauth_token(request)
+    try:
+        access_token = get_oauth_token(request)
+    except InvalidOAuthReturnError as err:
+        logger.error("InvalidOAuthReturnError. Remove invalid refresh_token and prompt for reauthentication.")
```

Comment:
This error message really doesn't contain any details that would be useful and would just be noise in the logs. Maybe it could at least include `request.user` incase we needed to try to debug an issue.

Reply:
Done. Added {request.user}. check [cca944e](https://github.com/tl-its-umich-edu/canvas-app-explorer/pull/211/commits/cca944e22591fdd7b13e2a1b2027f7714138fe22)